test/activerecord_test.rb

```diff
@@ -132,22 +132,24 @@ class SkipValidationsLambdaWithParamUser < ActiveRecord::Base
 end
 
 describe Enumerize::ActiveRecordSupport do
+  before do
+    User.delete_all
```

Comment:
otherwise some random tests would fail due to data inserted by `upset`. 
But as I see this was used it most tests explicitly anyway, so moved it here